sources/packages/backend/apps/queue-consumers/src/queue-consumers.module.ts

```diff
@@ -166,6 +170,8 @@ import { QueuesMetricsModule } from "./queues-metrics.module.module";
     CASActiveSupplierAndSiteFoundProcessor,
     CASKnownErrorsProcessor,
     CASSupplierSharedService,
+    StudentPdPpdReminderNotification,
```

Comment:
Please rename it to `StudentPDPPDReminderNotification` same for other named variables/methods along the PR.